<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParkingWindow</class>
 <widget class="QMainWindow" name="ParkingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ParkingWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="mainmenu">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>메인 메뉴</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parkingspace7">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>180</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>주차 가능</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parkingspace6">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>180</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>주차 가능</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parkingspace4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>주차 가능</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parkingspace5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>180</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>주차 가능</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parkingspace2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>주차 가능</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parkingspace1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>주차 가능</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parkingspace3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>주차 가능</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parkingspace9">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>주차 가능</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parkingspace10">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>370</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>주차 가능</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parkingspace13">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>370</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>주차 가능</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parkingspace12">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>370</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>주차 가능</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parkingspace11">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>370</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>주차 가능</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parkingspace8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>주차 가능</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parkingspace14">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>370</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>주차 가능</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="carNum">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>차량 번호</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>차량 번호를 입력하고 주차 가능 공간을 클릭해주세요</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>주차 하는법</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>차량 빼는 법</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>주차 공간을 클릭하고 차량 번호를 입력하세요</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
